wordpress-mobile/WordPress-iOS
# Jetpack Focus: Emphasize uninstalling the app in the migration success card

Fixes #20513 

## Description
Updates the migration success card design to make clear to users they should **uninstall the WordPress app** (not delete their site)

Design ref: qk5s4q4mZsXknwhMWo2vIY-fi-10230_21914

## How to test

Note that we're updating the value of existing localized strings. For testing purposes, change the keys by appending an extra letter to see the new value.
- `MigrationCenterViewConfiguration.swift`: L24, L27 
- `MigrationDeleteWordPressViewModel.swift`: L33, L38

1. Delete the Jetpack app
2. Install the WordPress app if not installed already
3. Build and run the Jetpack app
4. Follow the migration flow
5. ✅ The migration success card should match the new designs - please test light vs dark mode, landscape vs portrait, min / max font sizes
6. Tap on the migration success card
7. ✅ The overlay should match the new designs  - please test light vs dark mode, landscape vs portrait, min / max font sizes (note: we've decided to use ["on your device" instead of "on your iPhone"](p1681296746617889/1681290711.175429-slack-C0180B5PRJ4)) 

Before | After (light mode) | After (dark mode)
-- | -- | --
<img src="https://user-images.githubusercontent.com/6711616/231472148-7ee6b46b-b23d-4942-99a1-7e86d69a2101.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/231467540-9875cd0e-33e8-48e9-9e99-db400aeb866e.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/231467555-d5419951-cab0-4851-a9c4-fe6999c1dd00.png" width=200>
<img src="https://user-images.githubusercontent.com/6711616/231472156-55c6eeb3-0945-4b29-941c-1f760b577883.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/231467728-3bfcf1e3-bed6-4702-b899-7e013db5d975.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/231467734-51dfc74d-0ea6-461d-9a05-c3fbaf8b907e.png" width=200>

## Regression Notes
1. Potential unintended areas of impact
n/a

8. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

9. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
